lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -154,6 +154,10 @@ def check_alt_backend(alt_backend):
 @pytest.mark.parametrize("toolbar", ["toolbar2", "toolmanager"])
 @pytest.mark.flaky(reruns=3)
 def test_interactive_backend(backend, toolbar):
+    # Skipping the test for Travis-CI on arm64 platform
+    if os.environ.get('TRAVIS_CPU_ARCH') == "arm64":
```

Comment:
Does this need a default value (ie. `os.environ.get('TRAVIS_CPU_ARCH', '')` in case this environment variable isn't present?